src/main/java/randoop/reflection/OmitMethodsPredicate.java

```diff
@@ -152,4 +158,18 @@ boolean shouldOmit(final TypedClassOperation operation) {
   public String toString() {
     return "OmitMethodsPredicate: " + omitPatterns;
   }
+
+  /**
+   * Check if the given operation makes a call to hashcode and the class does not override hashcode.
+   *
+   * @param typedClassOperation
+   * @return true if the given operation calls hashcode but does not override it
+   */
+  private boolean makesCallToHashCode(TypedClassOperation typedClassOperation) {
+    CallableOperation callableOperation = typedClassOperation.getOperation();
```

Comment:
This method seems to always return false.  What is the point of the first 4 lines of the body?  Should the method be deleted?